### DISCUSSION TITLE
How custom p2 mirror？Thanks

### DISCUSSION THREAD

**Discussion by mozhuanzuojing** [eclipse-tycho/tycho#2222] [Q&A] [answered]
I am in China and want to build an eclipse plug-in project, how can I specify the P2 image address in China? I specified the mirror（`http://mirrors.ustc.edu.cn/eclipse/releases/2023-03/202303151000/`） in the target file，but it did not take effect. 

##### Screenshot
![image](https://user-images.githubusercontent.com/63572041/224548875-010393fe-bbef-4803-a954-f252b80f4e2f.png)

![image](https://user-images.githubusercontent.com/63572041/224548698-579eb715-47dd-4772-a919-797a0a4c57fb.png)

##### additional

this is my project https://gitee.com/mozhuanzuojing/jdec.jdt.ls.decompiler.git

**Comment by mickaelistria** [comment, marked as answer]
Try adding `-Dtycho.disableP2Mirrors` to your command-line to... disable p2 mirrors ;)
